### DISCUSSION TITLE
Song list in weeks only showing first two

### DISCUSSION THREAD

**Discussion by PlasmaOJ15** [KadeArchive/Kade-Engine#246] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/82743220/115133527-c05eb680-9fbd-11eb-8765-43957475adef.png)
![image](https://user-images.githubusercontent.com/82743220/115133539-d0769600-9fbd-11eb-9b22-97e38d1d9d59.png)
![image](https://user-images.githubusercontent.com/82743220/115133548-e7b58380-9fbd-11eb-8f7d-d5d0c10def53.png)
The songs list for the weeks seems to be fine, but both of them with only one song (Tutorial and Week 7) have nothing listed, and the others are missing their last song. The songs are still there, but dont show up in the week selection screen. Anyone know a fix?

**Comment by puyoxyz** [comment, marked as answer]
https://github.com/KadeDev/Kade-Engine/issues/251
